crates/openapi/src/routes/payment_method.rs

```diff
@@ -444,6 +444,35 @@ pub fn payment_method_session_list_payment_methods() {}
 )]
 pub fn payment_method_session_update_saved_payment_method() {}
 
+/// Payment Method Session - Delete a saved payment method
+///
+/// Delete a saved payment method from the given payment method session.
+#[cfg(feature = "v2")]
+#[utoipa::path(
+    delete,
+    path = "/v2/payment-method-session/:id",
+    params (
+        ("id" = String, Path, description = "The unique identifier for the Payment Method Session"),
+    ),
+    request_body(
+        content = PaymentMethodSessionDeleteSavedPaymentMethod,
+            examples(( "Update the card holder name" = (
+                value =json!( {
+                    "payment_method_id": "12345_pm_0194b1ecabc172e28aeb71f70a4daba3",
+                }
+            )
+        )))
+    ),
+    responses(
+        (status = 200, description = "The payment method has been updated successfully", body = PaymentMethodDeleteResponse),
```

Comment:
This description would have to be updated as well.

I'm guessing it would be:
```suggestion
        (status = 200, description = "The payment method has been deleted successfully", body = PaymentMethodDeleteResponse),
```